IMC_WebDev/FinalBlog.md

```diff
@@ -0,0 +1,224 @@
+## Building a Mini City: HTML and CSS Portion
+## Purpose
+HTML and CSS are extremely important parts of learning about how websites work. Every website uses HTML to display content such as text or images. Being able to grasp a few concepts of  HTML and CSS can help a lot when it comes to learning more intricate subjects in computer science, so it's important to get some experience with it!
```

Comment:
I like how you explain HTML in the intro for beginners. It would be helpful to also include a sentence about what CSS here, especially because the blog post is about both HTML and CSS

Reply:
Also, for the last sentence could use a different word choice instead of "important" to avoid redundancy. Maybe "crucial"/"critical"/"valuable"